.gitignore

```diff
@@ -29,8 +29,10 @@ dlldata.c
 .sconsign.dblite
 user_docs/*/*.html
 user_docs/*/*.css
+# TODO remove when migrating from t2t to md
+user_docs/*/*.md
 user_docs/*/keyCommands.t2t
-user_docs/build.t2tConf

```

Comment:
Unless there is an important reason not to, I'd suggest to keep this file ignored. While it no longer gets created developers would have it from previous builds, so their work tree will contain untracked files unnecessarily.